Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/DecryptFileViewModel.cs

```diff
@@ -119,6 +117,8 @@ protected override void InitializeModel()
                 .AddMenuProperty(Key, KeyInputMode.Key)
                 .AddMenuProperty(KeySecureString, KeyInputMode.SecureKey)
                 .BuildAndInsertMenuActions();
+
+            DecryptedFile.OrderIndex = propertyOrderIndex++;
```

Comment:
Is this really all we need to set here ? We should at least set explicity the IsPrincipal and IsVisible values.

Same for other similar occurrences ?